rpc/client/rpc_test.go

```diff
@@ -392,6 +392,20 @@ func TestNumUnconfirmedTxs(t *testing.T) {
 	mempool.Flush()
 }
 
+func TestCheckTx(t *testing.T) {
+	mempool := node.Mempool()
+
+	for _, c := range GetClients() {
+		_, _, tx := MakeTxKV()
+
+		res, err := c.CheckTx(tx)
+		require.NoError(t, err)
+		assert.Equal(t, abci.CodeTypeOK, res.Code)
```

Comment:
why not `require` here?

Reply:
I tend to use `require` only when the test cannot continue IF a certain condition is not true. In the above test, I check for err using require, because res is `nil` and the next line would panic if we continue.